src/shared/components/TableCellStatus.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+type TableCellStatusProps = {
+    status: "loading" | "error" | "na";
+    loadingAlt?:string;
+    errorAlt?:string;
+    naAlt?:string;
+};
+
+export default class TableCellStatus extends React.Component<TableCellStatusProps, {}> {
+    public render() {
+        let alt:string;
+        let text:string;
+        switch (this.props.status) {
+            case "loading":
+                alt=this.props.loadingAlt || "Querying server for data.";
+                text="LOADING";
```

Comment:
We don't need to do spinner for every column. I think they specifically want to see it as an annotation icon placeholder